app/models/user.rb

```diff
@@ -0,0 +1,5 @@
+class User < ApplicationRecord
+  validates :username, presence: true
+  validates :email, presence: true
+  validates :password, presence: true
```

Comment:
Please add the uniqueness validation to the username and email so that two users can't have the same data.